ui.tests/test-module/specs/recaptcha/recaptcha.runtime.spec.js

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
+ * Copyright 2023 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+describe("Form Runtime with Recaptcha Input", () => {
```

Comment:
we should have actual captcha ETE tests also.

Reply:
Recaptcha is rendered in iframe. To allow in test to access a cross origin iframe, I would have had to diasble web security in cy tests module. Please suggest.